apps/rails/app/services/wise/payout_api.rb

```diff
@@ -142,8 +142,15 @@ def get_transfer(transfer_id:)
     })
   end
 
-  # Wise::PayoutApi.new(wise_credential:).create_webhook(trigger: "transfers#state-change", url: "https://app.flexile.com/webhooks/wise/transfer_state_change")
-  # Wise::PayoutApi.new(wise_credential:).create_webhook(trigger: "balances#credit", url: "https://app.flexile.com/webhooks/wise/balance_credit")
+  # Example usage:
+  # Wise::PayoutApi.new(wise_credential:).create_webhook(
+  #   trigger: "transfers#state-change",
+  #   url: "https://#{ENV.fetch('APP_DOMAIN', 'app.flexile.com')}/webhooks/wise/transfer_state_change"
```

Comment:
```suggestion
  #   url: "https://#{ENV['APP_DOMAIN']}/webhooks/wise/transfer_state_change"
```
Don't think we need to account for this not being set

Reply:
Done